draft-ietf-rtcweb-jsep.xml

```diff
@@ -1165,10 +1165,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         bundle-only. This policy aims to minimize
                         candidate gathering and maximize multiplexing,
                         at the cost of less compatibility with legacy
-                        endpoints. When acting as answerer, if there
-                        if no bundle group in the offer, the
-                        implementation will reject all but the first
-                        m= section.</t>
+                        endpoints. When acting as answerer, the implementation
+                        will reject all but the first m= section and those
```

Comment:
how about

"will reject any m= sections other than the first m= section, unless they are in the same bundle group as that m= section."

or 

"will reject any m= sections other than the first m= section, unless they are bundled into that m= section."